src/controller.ts

```diff
@@ -60,7 +60,7 @@ export function delayCallback(callback: (...args: unknown[]) => boolean, ...args
     if (timers[callback.name])
         window.clearTimeout(timers[callback.name]);
 
-    timers[callback.name] = window.setTimeout(() => callback(...args), 100);
+    timers[callback.name] = window.setTimeout(() => callback(...args), 500);
```

Comment:
Why was this changed?

Reply:
I was running into some error during initialization when the delay was 100ms, but it seems to be gone now. should be able to ignore this change.